### DISCUSSION TITLE
Incorporating CSS Encapsulation in create-block Plugin Setup

### DISCUSSION THREAD

**Discussion by ZoomREI** [WordPress/gutenberg#62607] [Q&A]
Hello everyone,

I am working on a project using `@wordpress/create-block` tool. Can I get help integrating React-style CSS encapsulation to my build setup? I've been struggling trying to do this on my own through `package.json` and adding `webpack.config.js`

**Comment by stokesman** [comment]
> React-style CSS encapsulation

I assume your talking about an npm package. It might help for you to provide the actual package name and any errors your seeing.
